auth/src/args.rs

```diff
@@ -28,6 +28,11 @@ pub struct StartArgs {
     /// Stripe client secret key
     #[arg(long, default_value = "")]
     pub stripe_secret_key: String,
+
+    /// Auth JWT signing private key, as a base64 encoding of
+    /// a PEM encoded PKCS#8 v1 formatted unencrypted private key.
+    #[arg(long, default_value = "")]
```

Comment:
We might have discussed this in the past, but should we not have a default value for this?